src/ApiGateways/Mobile.Bff.Shopping/aggregator/Filters/AuthorizeCheckOperationFilter.cs

```diff
@@ -13,8 +13,10 @@ public class AuthorizeCheckOperationFilter : IOperationFilter
             public void Apply(Operation operation, OperationFilterContext context)
             {
                 // Check for authorize attribute
-                var hasAuthorize = context.ApiDescription.ControllerAttributes().OfType<AuthorizeAttribute>().Any() ||
-                                   context.ApiDescription.ActionAttributes().OfType<AuthorizeAttribute>().Any();
+                var hasAuthorize = context.MethodInfo.DeclaringType.GetCustomAttributes(true)
```

Comment:
Two comments here:

1. This is not the change recommended in the obsolescence message (suggests using `TryGetMethodInfo`) and
2. Replacing the OR `||` with a union hinders expression evaluation optimization.